### DISCUSSION TITLE
while pushing the files into git hub error is getting

### DISCUSSION THREAD

**Discussion by penna-obula** [community/community#105867] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

penna@ObulaReddy MINGW64 ~/OneDrive/Desktop/newrepo (master)
$ git push -u origin master
remote: Permission to penna-obula/newrepo.git denied to Susmitha-hub.
fatal: unable to access 'https://github.com/penna-obula/newrepo.git/': The requested URL returned error: 403

can any one suggest on ablove error please

**Comment by sapalazz** [comment]
Morning! I think you need to generate a new SSH key and add it to the ssh-agent.

Here's the full process, it's in the GitHub docs: [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).

Let me know if that works. :)
